src/Mono.Android/Test/Mono.Android-Test.Shared.projitems

```diff
@@ -9,41 +9,46 @@
+    <Compile Include="$(MSBuildThisFileDirectory)Android.Runtime\CharSequenceTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Android.Runtime\JavaCollectionTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Android.Runtime\JnienvArrayMarshaling.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Android.Runtime\XmlReaderPullParserTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Android.Runtime\XmlReaderResourceParserTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Android.Widget\AdapterTests.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Java.Interop\JavaConvertTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Java.Lang\ObjectArrayMarshaling.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Java.Lang\ObjectTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Java.Interop\JnienvTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\AppDomainTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\AssemblyInformationalVersionAttributeTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\ExceptionTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\TimeZoneTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System.IO\DriveInfoTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System.IO.Compression\GZipStreamTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System.Net\ProxyTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System.Net\SslTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System.Net\NetworkInterfaces.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System.Threading\InterlockedTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Java.Interop\JavaObjectExtensionsTests.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.Net\AndroidClientHandlerTests.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.Net\HttpClientIntegrationTests.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.RuntimeTests\MainActivity.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.RuntimeTests\MyIntent.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.RuntimeTests\NonJavaObject.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.RuntimeTests\NUnitInstrumentation.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.RuntimeTests\TestInstrumentation.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Android.OS\BundleTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Mono.Data.Sqlite\SqliteTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)..\..\Xamarin.Android.Build.Tasks\Tests\Xamarin.ProjectTools\Resources\Base\Image.9.png">
+      <Link>Resources\Drawable\Image.9.png</Link>
+    </AndroidResource>
```

Comment:
If we are adding this here do we still need https://github.com/xamarin/xamarin-android/blob/2ba0f5f3cc0d30ce751df3122971c65ea443059e/tests/Runtime-MultiDex/Mono.Android-TestsMultiDex.csproj#L76 ?

Reply:
No, I wouldn't think that we would need that.

Reply:
Removed.  Let's see if it still builds & runs! https://github.com/xamarin/xamarin-android/pull/2744/files#diff-53d0dd829aac4564989e7e33317ff2fcL74